resources/qml/MessageInput.qml

```diff
@@ -405,6 +410,7 @@ Rectangle {
             ToolTip.visible: hovered
             ToolTip.text: qsTr("Send")
             onClicked: {
+                messageInput.append(messageInput.preeditText)
```

Comment:
This doesn't work with the enter button, does it?

Reply:
well, enter already commits preedit text on the input methods I've tried, so I haven't given any thought to that. but if course that would differ depending on the input method. however, enter on a touchscreen keyboard shpuld just produce a new line, since shift+enter doesn't work and you'd more likely just use the send button anyway.